app/controllers/application_controller.rb

```diff
@@ -1,20 +1,29 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
-  rescue_from Notifications::Client::NotFoundError, with: :validation_notice_request_error
-  rescue_from Notifications::Client::ServerError, with: :validation_notice_request_error
-  rescue_from Notifications::Client::RequestError, with: :validation_notice_request_error
-  rescue_from Notifications::Client::ClientError, with: :validation_notice_request_error
-  rescue_from Notifications::Client::BadRequestError, with: :validation_notice_request_error
-
   before_action :find_current_local_authority_from_subdomain
   before_action :prevent_caching
   before_action :set_current_user
+  before_action :configure_permitted_parameters, if: :devise_controller?
 
   attr_reader :current_local_authority
 
   helper_method :current_local_authority
 
+  def after_sign_in_path_for(resource)
+    if session[:mobile_number] && !resource.mobile_number?
+      resource.assign_mobile_number!(session[:mobile_number])
+      session.delete(:mobile_number)
+    end
+
+    respond_to do |format|
+      format.html do
+        flash[:notice] = "Signed in successfully."
+        root_path
```

Comment:
redirect_to root_path, notice: "Signed in successfully."

Reply:
Yeah I wanted to do this but I get
```
AbstractController::DoubleRenderError:
   Render and/or redirect were called multiple times in this action.
```